spec/integration/status_updates_spec.rb

```diff
@@ -2,7 +2,7 @@
   let(:reference) { "b6589b2b-8f8e-457b-9ddf-237b62438ad1" }
 
   let!(:delivery_attempt) do
-    create(:delivery_attempt, reference: reference, status: "sending")
+    create(:delivery_attempt, id: reference, status: "sending")
```

Comment:
I'm not sure exactly why we've got a hardcoded uuid above so I may be misunderstanding something but could this just be a `let!(:delivery_attempt) { create(:delivery_attempt, status: "sending") }` and then reference is replaced with `delivery_attempt.id` ?

Reply:
Yep, agreed that seems unnecessary... I've fixed that now!